subprojects/soak/src/integTest/groovy/org/gradle/vfs/VirtualFileSystemRetentionSoakTest.groovy

```diff
@@ -90,11 +90,9 @@ class VirtualFileSystemRetentionSoakTest extends DaemonIntegrationSpec implement
             daemon.assertIdle()
             assertWatchingSucceeded()
             boolean overflowDetected = detectOverflow(daemon, endOfDaemonLog)
-            int expectedNumberOfRetainedFiles = retainedFilesInLastBuild - numberOfChangesBetweenBuilds
-            int retainedFilesAtTheBeginningOfTheCurrentBuild = retainedFilesSinceLastBuild
-            if (overflowDetected) {
-                assert retainedFilesAtTheBeginningOfTheCurrentBuild == 0
-            } else {
+            if (!overflowDetected) {
+                int expectedNumberOfRetainedFiles = retainedFilesInLastBuild - numberOfChangesBetweenBuilds
+                int retainedFilesAtTheBeginningOfTheCurrentBuild = retainedFilesSinceLastBuild
```

Comment:
Is this rename useful? I'd expect IntelliJ to flag it to be simplified.

Reply:
It is useful since `retainedFilesSinceLastBuild` needs to parse the output of the build and is in two assertions.

Reply:
But it is just a different name for `retainedFilesSinceLastBuild`. Not a big deal, happy to merge it, too, just that I don't get it. :)